libcontainer/configs/validate/validator_test.go

```diff
@@ -82,6 +82,23 @@ func TestValidateHostname(t *testing.T) {
 	}
 }
 
+func TestValidateDomainname(t *testing.T) {
```

Comment:
Same for tests -- maybe it's better to unify the test cases for hostname and domainname, putting a few cases into the same test. Otherwise we just repeat many lines of code without adding much value.